GetterAndSetter.metta

```diff
@@ -0,0 +1,82 @@
+! (register-module! ../metta-attention)
+! (import! &self types)
+
+ ;############################ General Overview  #####################
+
+ ; set_av function accepts a pattern/node with the attention values
+ ; and set the attention value to the pattern/node.
+ ;
+ ; i.e !(set_av (link a b) (400 400 1))
+ ;
+ ; set_stv function accepts a pattern/node with the truthvalues
+ ; and set the simple truth value to the pattern/node.
+ ;
+ ; i.e !(set_stv (link a b) (0.1 0.9))
+ ;
+ ; get_av function accepts a pattern/node and returns the attention
+ ; value attached to the pattern/node.
+ ;
+ ; i.e !(get_av (link a b))  output: (AV 400 400 1)
+ ;
+ ; get_stv function accepts a pattern/node and returns the simple truth
+ ; value attaced to the pattern/node
+ ;
+ ; i.e !(get_stv (link a b))  output: (STV 0.1 0.9)
+
+ ;############################# End ###################################
```

Comment:
Next time it would be great if add the comment above each functions instead of having all together. like the following 

;; some description what the function do
----function implementation

;; some description what the function do
----function implementation